Source/SpiderEye.Windows/Interop/KeyMapper.cs

```diff
@@ -36,6 +36,8 @@ private static Keys MapModifier(ModifierKey modifier)
                     return Keys.Control;
 
                 case ModifierKey.Super: // there is Keys.LWin but it's not a modifier key
+                    return Keys.Control;
```

Comment:
According to the xml doc of ModifierKey.Super this should be the windows key on windows...

Reply:
Fixed